Fix for #30 Documentation for Flogo rules (#41)
project-flogo/rules: README.md

$ ./$GOPATH/bin/rulesapp

```
## Running Rules in a Flogo App

> To use the Rules API in a Flogo App, refer to `examples/flogo/simple/README`

Reviewer: ```suggestion
To use the Rules action in your Flogo App, refer to `examples/flogo/simple/README`
```
